setup.cfg

```diff
@@ -86,9 +85,8 @@ install_requires =
     blinker
     cached_property~=1.5;python_version<="3.7"
     cattrs~=1.1, !=1.7.*
-    # Required by vendored-in connexion
-    clickclick>=1.2
     colorlog>=4.0.2, <7.0
+    connexion[swagger-ui,flask]>=2.10.0
```

Comment:
What does the flask extra here do?

Reply:
Good question - that's why I need some coments from @mik-laj  and @ephraimbuddy @uranusjr. 

I just added it back from the original connexion vendoring commit: https://github.com/apache/airflow/commit/bb115da0b78036ace5e31e2139c12ca044b3a97d#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52L90 

My guess it's needed to simply integrate connexion routes with flask.

Reply:
`swagger-ui` is required for docs in WebUI.
https://github.com/apache/airflow/blob/7e29506037fa543f5d9b438320db064cdb820c7b/airflow/www/extensions/init_appbuilder_links.py#L31-L36
`flask` is required, because connexion supports two. framework: `aiohttp` and `flask`, so we need to decide.

Reply:
From I can tell from the source code, the only thing the `flask` extra does is to install Flask, and since Airflow already requires Flask, this extra is essentially no-op. Keeping it is fine IMO since it serves as documentation how we expect to use `connexion`.